estate/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    'name': "Estate",
+    'version': '1.01',
```

Comment:
We start at 1.0, which is the default value in manifest so you can even omit it here :slightly_smiling_face: